File: sem-1/wdp-zadania/lab9/zad3.py
def sortByInsert(arr: list[int]):
    j = len(arr) - 1
    while j >= 0:
        p = arr[j]
        i = j + 1
        while i < len(arr) and p > arr[i]:
            arr[i - 1] = arr[i]
            i += 1
        arr[i - 1] = p
        j -= 1
    return arr


print(sortByInsert([8, 3, 12, 4, -5, 10, 9, 8]))

File: sem-2/jp1/lab7/zad3.scala
package lab7

def position[A](arr: List[A], element: A): Option[Int] = {
  val helper = arr.indexOf(element)
  if helper >= 0 then Option(helper)
  else None
}

@main
def lab7zad3(): Unit = {
  val myList = List(2, 1, 1, 5)
  println(position(myList, 1))
  println(position(myList, 3))
}